pkg/apis/pravega/v1beta1/pravegacluster_types_test.go

```diff
@@ -111,14 +112,105 @@ var _ = Describe("PravegaCluster Types Spec", func() {
 			Ω(p.Spec.TLS.IsSecureController()).To(Equal(false))
 		})
 	})
-	Context("checking volume claim template is present", func() {
+	Context("Checking various parameters", func() {
 		BeforeEach(func() {
 			p.Spec.Version = "0.6.0"
 			p.WithDefaults()
 		})
 		It("should set volume claim template", func() {
 			Ω(p.Spec.Pravega.CacheVolumeClaimTemplate).ShouldNot(BeNil())
 		})
+
```

Comment:
Not seen ut added for waitforclusterTerminate fn

Reply:
added